src/index.js

```diff
@@ -51,8 +51,8 @@ export default function Navaid(base, on404) {
 
 		function click(e) {
 			var y, x = e.target.closest('a');
-			if (!x || !x.href || x.target || x.host !== location.host) return;
-			if (e.ctrlKey || e.metaKey || e.altKey || e.shiftKey || e.button) return;
+			if (!x || !x.href || x.target || x.host !== location.host) return;
```

Comment:
My IDE indicated this space was a bad character. It was `\u00a0` instead of a regular space `\u0020`. Didn't seem to hurt anything, but thought I would fix it anyway.